src/views/Main/Main.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+import BlogCard from '../../components/BlogCard/BlogCard';
+import getBlogs from '../../services/Blogs';
+import './Main.css';
+
+export default function Main() {
+  const [blogs, setBlogs] = useState([]);
+  const [loading, setLoading] = useState(true);
+  useEffect(() => {
+    const fetchData = async () => {
+      const resp = await getBlogs();
+      setBlogs(resp);
+      setTimeout(() => {
+        setLoading(false);
+      }, 750);
```

Comment:
Just an FYI, when using setTimeout in the future, we will need to call this function to clear the timeout so an error doesn't occur when we begin navigating through our applications :) 
const timer = setTimeout(() => {
          setLoading(false);
        }, 3000);
        return () => {
          clearTimeout(timer);
        };